lib/facade/FileClient.ts

```diff
@@ -79,6 +79,15 @@ export class FileClient {
     if (params) {
       return await this.client.decrypt(params);
     }
+
+    //@ts-ignore
+    const stream = source?.stream;
```

Comment:
<img width="301" alt="image" src="https://user-images.githubusercontent.com/41641307/186143028-9421a625-ba66-4a39-8b3f-9cc11bea93f5.png">
If its not PlainTextstream it will be undefined. And decrypt can use other sources.
And remove ts-ignore, it was probably helping you by highlighting that